server/integration-tests/src/test/java/io/smallrye/graphql/tests/client/dynamic/NestedRecordsTest.java

```diff
@@ -69,9 +68,13 @@ public void testNestedRecordWithMissingFieldInQuery() throws Exception {
         }
     }
 
-    @Ignore("In Java 21 this test fails due to a bug (presumably) in the JSON-B implementation – Yasson)")
     @Test
     public void testNestedRecordWithListWithMissingFieldInQuery() throws Exception {
+        if (System.getProperty("java.version").startsWith("21.")) {
```

Comment:
This affects 22 too, could we make it safer and do something along the lines of
```
String major = Integer.valueOf(System.getProperty("java.version").split("\\.")[0]);
if(major >= 21 && major <= 22) {
```
? Also, when skipping a test, print an info message into the log

Reply:
yea could do